front/php/templates/footer.php

```diff
@@ -54,7 +54,7 @@
 
 <!-- NetAlertX -->
 <script src="js/handle_version.js"></script>
-<script defer src="js/ui_components.js?v=<?php include 'php/templates/version.php'; ?>"></script>
```

Comment:
FYI - I had to reinsert the defer statement because the devices listing page wouldn't load at all - happy to investigate the detail page further, but for now I had to roll back this change

Reply:
Interesting. For me, with the 'defer' the details page loads but all fields are empty.
If I delete the 'defer', the page loads with all details.

Reply:
Is it possible that it's caused by the values in the fields? I'm running a clean installation in parallel and the details page loads fine. If recommend checking the dev console of the browser.